Resolved misc. Cordova UI Issues

SailingSteve opened the pull request:
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
Reworked breakpoints for iPads.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Deleted styled-them.js, it was not being used.
9/3/22: Another pass over header sizing for tablets.
9/8/22: Reworked DeviceDialog for Cordova
Do not change "u-show-desktop from 991px to 740px." for Cordova, see displayNoneIfSmallerThanDesktop()
Moved header-logo-img into a styled div
Moved header-nav__avatar into LazyImage.jsx and removed in from main.css

SailingSteve committed:
Resolved misc. Cordova UI Issues
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Reworked breakpoints for iPads.
Deleted styled-them.js, it was not being used.

SailingSteve committed:
Resolved misc. Cordova UI Issues
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Reworked breakpoints for iPads.
Deleted styled-them.js, it was not being used.

SailingSteve committed:
Resolved misc. Cordova UI Issues
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Reworked breakpoints for iPads.
Deleted styled-them.js, it was not being used.

SailingSteve committed:
Resolved misc. Cordova UI Issues
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Reworked breakpoints for iPads.
Deleted styled-them.js, it was not being used.
Reworked DeviceDialog for Cordova
Do not change "u-show-desktop from 991px to 740px." for Cordova
Moved header-logo-img into a styled div
Moved header-nav__avatar into LazyImage.jsx and removed in from main.css

SailingSteve committed:
Resolved misc. Cordova UI Issues
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Reworked breakpoints for iPads.
Deleted styled-them.js, it was not being used.
Reworked DeviceDialog for Cordova
Do not change "u-show-desktop from 991px to 740px." for Cordova
Moved header-logo-img into a styled div
Moved header-nav__avatar into LazyImage.jsx and removed in from main.css

SailingSteve commented:
@DaleMcGrew Ok, this is ready to go

DaleMcGrew commented on src/css/main.css:
```diff
@@ -11533,19 +11533,13 @@ h1, h2, h3, h4, h5, h6 {
   }
 }
 
-@media all and (max-width: 991px) {
```
Hi @SailingSteve u-show-desktop is still used in many places and unless I'm mistaken, this CSS can't be removed.

SailingSteve replied:
No, all the u-show-desktop references are gone

DaleMcGrew replied:
src/js/components/Navigation/HeaderBar.jsx line 518

SailingSteve replied:
Ok, fixed that one.

DaleMcGrew commented on src/js/components/Navigation/Header.jsx:
```diff
@@ -584,3 +585,8 @@ Header.propTypes = {
   // pathname: PropTypes.string,
 };
+
+const BackToSettingsMobileDesktopSpan = styled('div')`
```
This might want to be a span instead of div.

SailingSteve replied:
Ok

DaleMcGrew commented on src/js/components/SetUpAccount/AddContactsFromGoogleButton.jsx:
```diff
@@ -235,6 +243,25 @@ class AddContactsFromGoogleButton extends Component {
     });
   }
 
+  cordovaSignOn () {
```
My hope is to use an entirely different component when on a device, in most cases, but there might be some scenarios in the future where we also want to support importing from Gmail. But I consider this a low priority.

SailingSteve replied:
Those changes were to support Sign In With Google in Cordova, which would take weeks more to finish.  As is, they don't hurt anything, and took a day+ to figure out.  I would only copy a class to make a Cordova version as a last resort.

SailingSteve committed:
Resolved misc. Cordova UI Issues
Made the top menu larger (and more consistent) in both Cordova and WebApp for tablets.
iPad mini, gets the top menu now.
Improved Enter Your Address modal layout in WebApp and Cordova.
Reworked breakpoints for iPads.
Deleted styled-them.js, it was not being used.
Reworked DeviceDialog for Cordova
Do not change "u-show-desktop from 991px to 740px." for Cordova
Moved header-logo-img into a styled div
Moved header-nav__avatar into LazyImage.jsx and removed in from main.css
Fixed one last u-show-desktop
Made BackToSettingsMobileDesktopSpan into a span instead of a div

SailingSteve commented:
Ok, all these have been resolved.

DaleMcGrew commented:
👍
